### ISSUE TITLE
"Array to string conversion" error when opening (empty) Collection

### ISSUE THREAD

**Issue by MrMooky** [statamic/cms#6326]
### Bug description

When opening a collection overview, I suddenly get the following error:

> Array to string conversion (View: /x/y/vendor/statamic/cms/resources/views/collections/empty.blade.php).

 I had the same error when there were entries in the collection, but only with `show.blade.php` instead of `empty.blade.php`. 

Cleared all caches, dumped autoload... error is still there. 

### How to reproduce

Not really sure. The collection is empty now and there is nothing I can think of. This is the config for the collection:

```
title: News
sites:
  - default
  - en
propagate: false
template: cms/news/show
layout: layout
revisions: false
route: 'news/{slug}'
date: true
sort_dir: asc
date_behavior:
  past: public
  future: private
```

### Logs

```shell
https://flareapp.io/share/V7jLaKJm#F80
```

### Versions

Statamic 3.3.18 Pro
Laravel 9.19.0
PHP 8.1.4
jonassiewertsen/statamic-livewire 2.9.0
octoper/statamic-blade-components 2.0.0

### Installation

Existing Laravel app

### Antlers Parser

runtime (new)

**Comment by jasonvarga** [comment]
This has something to do with that livewire package. Could you open an issue there instead?

**Comment by MrMooky** [comment]
I just removed both addons, cleared all caches and the error remains.

```
Statamic 3.3.19 Pro
Laravel 9.19.0
PHP 8.1.4
No addons installed
```

The stacktrace is identical. I'm not using Livewire anywhere in the backend myself and the error appears when opening the route `Statamic\Http\Controllers\CP\Collections\CollectionsController@show`.

Now here is the fun part: I also completed removed Livewire from composer, cleared caches, dumped composer autoload and that error is still pointing to Livewire. o_O I also debugged the source in the vendor folder.

The line `@section('title', Statamic::crumb($collection->title(), 'Collections'))` is causing the error, but only in the `News` collection. Other collections work just fine, for whatever reason.

<img width="723" alt="Screenshot 2022-07-13 at 08 51 15" src="https://user-images.githubusercontent.com/3603346/178670018-217fc7b3-7e3f-4220-a795-fca98ebbd467.png">

The function `crumb` expects and array (?) it seems, but it getting a string, either "News", or "Projects" in my case.

```
public static function crumb(...$values)
{
    return implode(' ‹ ', array_map('__', $values));
}
```

Not sure though why Livewire is still in the picture.
